Microsoft.Toolkit.Uwp.SampleApp/SamplePages/StringExtensions/StringExtensionsPage.xaml.cs

```diff
@@ -26,32 +25,13 @@ public StringExtensionsPage()
             this.InitializeComponent();
         }
 
-        protected override void OnNavigatedTo(NavigationEventArgs e)
+        private void OnTextChanged(object sender, TextChangedEventArgs e)
```

Comment:
Currently, `TextBlock`s are not displayed before typing in `TextBox`. So call this method after `InitializeComponent` or in `Loaded` event.

Reply:
Updated as you think this is important. My preference is to not show results until there's any input but I'll defer to your preference here.